website/docs/r/muting_schedule.html.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: 'sumologic'
+page_title: 'SumoLogic: sumologic_muting_schedule'
+description: |-
+  Provides the ability to create, read, delete, and update muting schedule.
+---
+
+# sumologic_muting_schedule
+
+Provides the ability to create, read, delete, and update [MutingSchedule][1].
+
+## Example One Time Muting Schedule For All monitor
+
+```hcl
+resource "sumologic_muting_schedule" "muting_schedule" {
+  name = "Muting Schedule For one time"
+  description = "This is an example for one time Muting schedule for all monitor"
+	type = "MutingSchedulesLibraryMutingSchedule"
+	content_type = "MutingSchedule"
+	monitor {
+		all = true
+	  }
+	schedule  {
+		timezone = "America/Los_Angeles"
+		start_date = "2023-08-05"
+		start_time = "00:00"
+		duration = 40
+	}
+}
+```
+
+## Example One Time Muting Schedule For Specifc Monitor/Folder ids
```

Comment:
not: `One-time Muting Schedule at ... for Specific Monitor/Folder IDs`

Reply:
updated.